src/first_steps/intro.md

```diff
@@ -26,6 +31,15 @@ There are a lot of settings that customize Rizin's behaviour, see them with el.
 You can save your preferred settings in ~/.rizinrc.
 ```
 
+::: {.callout-tip}
```

Comment:
is this compatible with quarto?

Reply:
This is what they document at least: https://quarto.org/docs/authoring/callouts.html